4-WebApp-your-API/4-2-B2C/TodoListService/appsettings.json

```
@@ -3,10 +3,10 @@
     "Instance": "https://fabrikamb2c.b2clogin.com",
     "ClientId": "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6",
     "Domain": "fabrikamb2c.onmicrosoft.com",
-    "SignedOutCallbackPath": "/signout/B2C_1_susi",
-    "SignUpSignInPolicyId": "b2c_1_susi",
-    "ResetPasswordPolicyId": "b2c_1_reset",
-    "EditProfilePolicyId": "b2c_1_edit_profile" // Optional profile editing policy
+    "SignedOutCallbackPath": "/signout/B2C_1_susi_reset_v2",
+    "SignUpSignInPolicyId": "B2C_1_susi_reset_v2",
+    "ResetPasswordPolicyId": "B2C_1_susi_reset_v2",
```

```suggestion
    "ResetPasswordPolicyId": "B2C_1_susi_reset_v2",
```

I think this can be removed now, since its not a separate user-flow. In the README, this should be mentioned somewhere (that the new SUSI user-flow is capable of handling password reset flows)

---

Updated config and Readme file.